docs/source/cuda._sanitizer.rst

```diff
@@ -0,0 +1,95 @@
+.. currentmodule:: torch.cuda._sanitizer
+
+torch.cuda._sanitizer
+=====================
+
+Overview
+--------
+
+.. automodule:: torch.cuda._sanitizer
+
+
+Usage
+------
+
+Here is an example of a simple synchronization error in PyTorch:
+
+::
+
+    import torch
+
+    a = torch.rand(4, 2, device="cuda")
+
+    with torch.cuda.stream(torch.cuda.Stream()):
+        torch.mul(a, 5, out=a)
+
+The ``a`` tensor is initialized on the default stream and, without any synchronization
+methods, modified on a new stream. When this script is run on the commandline with:
```

Comment:
Explain why this is a bug. Say it clearly: the two kernels will run concurrently on the same data, thus the second kernel might read some uninitialized data before the first kernel was able to write it, or the first kernel might even overwrite some data of the second kernel.

Reply:
Done.